frontend/src/metabase/ui/components/inputs/index.ts

```diff
@@ -4,6 +4,7 @@ export * from "./DateInput";
 export * from "./DatePicker";
 export * from "./FileInput";
 export * from "./Input";
+export * from "./MultiSelect";
```

Comment:
Using the same value twice results in the following React warning:

```
Warning: Encountered two children with the same key, `9`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

![image](https://github.com/metabase/metabase/assets/6830683/ca1c49bb-c80c-4667-b5c4-54c324b44872)

Previously it was not possible to enter the same value twice.

Reply:
https://github.com/metabase/metabase/issues/36748